packages/flutter_markdown/lib/src/builder.dart

```diff
@@ -470,9 +470,27 @@ class MarkdownBuilder implements md.NodeVisitor {
         current.children.add(_buildRichText(const TextSpan(text: '\n')));
       } else if (tag == 'th' || tag == 'td') {
         TextAlign? align;
+        // `style` was using in pkg:markdown <= 6.0.1
+        // Can be removed when min pkg:markedwn > 6.0.1
         final String? style = element.attributes['style'];
         if (style == null) {
-          align = tag == 'th' ? styleSheet.tableHeadAlign : TextAlign.left;
+          // `align` is using in pkg:markdown > 6.0.1
+          final String? alignAttribute = element.attributes['align'];
+          if (alignAttribute == null) {
+            align = tag == 'th' ? styleSheet.tableHeadAlign : TextAlign.left;
+          } else {
```

Comment:
I added a `_newMarkdown` flag in the test. We might-could do that here, too!

To discuss...

Reply:
I’d prefer to bump minimum Markdown version and simplify the tests.

Reply:
I *totally* understand. We'd like to do a major bump, but there is still more to do. And we can't roll to google3 unless things are passing.